polaris/utils/types.py

```diff
@@ -24,19 +24,26 @@
 The second item can either be a single test set or a dictionary with multiple, named test sets.
 """
 
-PredictionsType: TypeAlias = Union[np.ndarray, dict[str, Union[np.ndarray, dict[str, np.ndarray]]]]
+ListOrArrayType: TypeAlias = list | np.ndarray
```

Comment:
Do we currently host benchmarks or competitions (or will we soon) that expect predictions of a type other than `int`? If not, we could tighten the typing to `list[int]`?

Reply:
Yes, we do! For example, in https://github.com/polaris-hub/polaris/pull/199 the prediction is a 3D molecule pose.